docs/index.html

```diff
@@ -103,8 +103,8 @@ <h1 class="logo">
               </select>
             </li>
             <li class="fine-grained-options__option">
-              <select class="settings__option settings__option--select js-settings-option" id="level_1_keep_special_comments" name="level[1][keepSpecialComments]">
-                <option value="*" selected>Special comments: keep all</option>
+              <select class="settings__option settings__option--select js-settings-option" id="level_1_keep_special_comments" name="level[1][specialComments]">
```

Comment:
@ngyikp could you change id to `level_1_special_comments` to keep everything consistent?